### ISSUE TITLE
Mission control won't change from metric system to imperial 

### ISSUE THREAD

**Issue by Eg681** [iNavFlight/inav-configurator#2108]
I've tried changing from metric units to imperial in mission control inav 6.1.1 but stays on metric system,advanced tuning page and osd changing just fine, I would like to enjoy some waypoints flying but very frustrating converting cm and m to feet mainly for the altitude. Thanks

**Comment by breadoven** [comment]
Units don't apply in Mission Control from what I remember. Something that needs adding though for sure.

**Comment by Eg681** [comment]
Yeah Mr D said same thing so I guess we will have to wait, I have inav on 5 different planes but I won't use mission control. To much math and alot of room for errors.
